src/commons/components/catalog-info/catalog-info.tsx

```diff
@@ -4,7 +4,12 @@ import React from 'react';
 import './catalog-info.css';
 import './catalog-info-media.css';
 
-function CatalogInfo(props){   
+interface CatalogInfoProps {
+  activeGenre:string,
+  activeCategory:string
+}
+
+function CatalogInfo(props: CatalogInfoProps){
```

Comment:
fix it

Example:
const CatalogGenres: React.FunctionComponent<CatalogGenresProps> = ({genres, isActiveGenre, genreClick}) => {